src/03_modules.py

```diff
@@ -10,22 +10,27 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
+print(sys.argv)
 
 # Print out the OS platform you're using:
 # YOUR CODE HERE
+print(sys.platform)
 
 # Print out the version of Python you're using:
 # YOUR CODE HERE
-
+print(sys.version)
 
 import os
 # See the docs for the OS module: https://docs.python.org/3.7/library/os.html
 
 # Print the current process ID
 # YOUR CODE HERE
+print(os.geteuid())
```

Comment:
This problem wants you to grab the current process ID and os.geteuid() grabs you the current process' effective user id. What that means exactly I don't know, but you should be using os.getpid(). 😄